ddtrace/contrib/kafka/patch.py

```diff
@@ -184,7 +205,15 @@ def traced_poll(func, instance, args, kwargs):
             message_key = message.key() or ""
             message_offset = message.offset() or -1
             span.set_tag_str(kafkax.TOPIC, message.topic())
-            span.set_tag_str(kafkax.MESSAGE_KEY, ensure_text(message_key, errors="replace"))
+
+            # If this is a deserializing consumer, do not set the key as a tag since we
+            # do not have the serialization function
+            if (
+                not isinstance(instance, _DeserializingConsumer)
```

Comment:
This class can be None too
?
```suggestion
                _DeserializingConsumer is not None and not isinstance(instance, _DeserializingConsumer)
```